xscen/regridding.py

```diff
@@ -146,7 +146,9 @@ def regrid(
 
     # Attrs
     out.attrs["cat/processing_level"] = to_level
-    out.attrs["cat/domain"] = ds_grid.attrs["cat/domain"]
+    out.attrs["cat/domain"] = (
```

Comment:
We'll have to do that change elsewhere too. I know that `indicators.compute_indicator` crashes for that same reason if `cat/variable` is not there...

Reply:
J'ai fait le tour des "cat/" et modifié 2 endroits  pour qu'ils ne fassent plus planter la fonction en leur absence.